src/sparseml/exporters/transforms/kv_cache/configs.py

```diff
@@ -84,7 +81,7 @@ class Config:
 
 OPT_CONFIG = KeyValueCacheConfig(
     model_name="opt",
-    positions_adjustment_transform=PositionsAdjustmentOPT,
+    additional_transforms=AdditionalTransformsOPT,
```

Comment:
would be great to support a list here eventually instead in case we don't want to squash many steps into a single transform or enable more code sharing between model specific additional transforms outside of inheritance